### PR TITLE
refactor(prompts): alphabetize list options

### PR DESCRIPTION
**Pull request by kingcody**

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -90,12 +90,13 @@ var AngularFullstackGenerator = yeoman.generators.Base.extend({
...
           name: 'script',
+          default: 1,
           message: 'What would you like to write scripts with?',
           choices: [ 'JavaScript', 'CoffeeScript'],
           filter: function( val ) {
             var filterMap = {
-              'JavaScript': 'js',
-              'CoffeeScript': 'coffee'
+              'CoffeeScript': 'coffee',
+              'JavaScript': 'js'
```
Comment (Awk34) [inline app/index.js]: Is there a point to alphabetical sorting? I think it makes more sense to have JS first at least.

**Comment by kingcody** [comment]
Well since we have generator defaults set, such as `javascript` over `coffee`, the alphabetizing is for readability and a consistent natural sorting order. Anyways, that's my opinion on it.

**Comment by Awk34** [comment]
I'm fine with the style changes and OAuth ordering, but I feel like JS should precede CoffeeScript. I think it makes more sense to have the simpler, non-transpiled language first, and then the more complicated options after. Same with CSS, but I think the default should follow right after the non-tranpiled language.

**Comment by kingcody** [comment]
Speaking of, why do we default to `sass` instead of `css`?

**Comment by Awk34** [comment]
I believe it's the most popular option. I'd like to change the language options to the following too:

```
JavaScript
JavaScript + Babel
CoffeeScript
```

**Pull request closed without merging**